test000.c: 函数 `main' :
test000.c:4: エラー: `9223372036854775808' が `long long int' の範囲を超えています.
test000.c:5: エラー: `0xfffffffffffffffff' が `unsigned long long int' の範囲を超えています.
test000.c:6: エラー: `0x10000000000000000' が `unsigned long long int' の範囲を超えています.
test000.c:7: エラー: `02000000000000000000000' が `unsigned long long int' の範囲を超えています.
